### ISSUE TITLE
wrong rendering for span p-author

### ISSUE THREAD

**Issue by jodumont** [indieweb/wordpress-uf2#41]
After activating the last version (1.1.0) of your plugin on WordPress 4.9.8 with [understrap theme](https://github.com/understrap/understrap)
my meta entry become : ` Posted on September 12, 2018 by <span class='p-author h-card'>Author</span> `
instead of : ` Posted on September 12, 2018 by Author`
for fun I look into the code and I think the error is [here](https://github.com/indieweb/wordpress-uf2/blob/66fef5f6236cc52446f5fe9fd8b70eae471842fb/includes/genesis.php#L71)
but since I'm totally not a PHP developer I'll pass on the potential PR

**Comment by pfefferle** [comment]
This is a problem if themes escape the output... the problem is not the code you referenced, but here: https://github.com/indieweb/wordpress-uf2/blob/66fef5f6236cc52446f5fe9fd8b70eae471842fb/includes/class-uf2-author.php#L20

I am working on a different solution, that should solve the problem in the future: https://github.com/indieweb/wordpress-mf2-feed

For now I would recommend to diable the plugin. I will also try to remove it from the IndieWeb plugin.

**Comment by pfefferle** [comment]
@miklb feel free to take over the plugin.

**Comment by miklb** [comment]
I’ve already forked it.

Cheers,

Michael

**Comment by pfefferle** [comment]
Why not taking over this plugin? You have admin rights and you can release new versions by simply tagging a current state. Or what is the reason to give a thumb down on my comment?

**Comment by miklb** [comment]
I’m leaving the IndieWeb WP GH group. I don’t want to be part of the GitHub org.

**Comment by miklb** [comment]
I haven't been able to revisit my fork lately https://github.com/miklb/wordpress-uf2 but if you can open the issue there I will compare against the changes I've made there.

**Comment by jodumont** [comment]
@miklb sorry I don't understand; we are discussing under an open issue ?

**Comment by miklb** [comment]
No. I believe this repo has been deprecated but I am still working on a fork & have some changes in it already. If you want to follow up there I can take a look in a day or two.

**Comment by jodumont** [comment]
@miklb thank to following with me and trying to help me, but I'm sorry; I think I miss understood you where I should follow, because on [your repo](https://github.com/miklb/wordpress-uf2) the issues features is not activated.

+ as you probably could tell I'm not a wordpress dev
but I compare SemPress and understrap (my theme) than this is the big difference I could tell might make the escaping

### SemPress theme use : **printf**
`<h1 class="page-title"><?php printf( __( 'Author Archives: %s', 'sempress' ), '<a class="url u-url fn p-fn n p-name" href="' . get_author_posts_url( get_the_author_meta( 'ID' ) ) . '" title="' . esc_attr( get_the_author() ) . '" rel="me author" itemprop="url"><span itemprop="name">' . get_the_author() . '</span></a>' ); ?></h1>`

### while understrap theme use : **php esc_html_e**
`<h1><?php esc_html_e( 'About:', 'understrap' ); ?><?php echo esc_html( $curauth->nickname ); ?></h1>`

BTW if you have test you want I do; I'm willing to learn and do

**Comment by jodumont** [comment]
FYI: this issue also appear with the **Independent Publisher theme** if you enable **Multi-Author Mode Option**
